x/gamm/breaking_changes_notes.md

```diff
@@ -4,8 +4,8 @@
 
 * PoolAssetsQuery
   * Deleted
-  * New query is TotalLiquidity (TODO: Write this query)
-  * Or if you wanted the pool weights, you must now query the pool itself.
+  * New query is TotalLiquidity
+  * If you wanted the pool weights, you must now query the pool itself.
     * Please give feedback if more queries should be exposed that are pool-type specific
 * QuerySpotPrice
   * The `withswapfee` param is now removed. If this was needed for anything, please flag it. Its mainly removed due to not having a clear use, and a better query can probably be crafted for.
```

Comment:
* Rename TokenInDenom to QuoteAssetDenom
* Rename TokenOutDenom to BaseAssetDenom

Dilan from Imperator pointed out this these might be switched